cli/tools/compile.rs

```diff
@@ -364,17 +379,6 @@ fn resolve_root_dir_from_specifiers<'a>(
       }
     }
   }
-  let found_dir = if is_file_system_root(found_dir) {
-    found_dir
-  } else {
-    // include the parent dir name because it helps create some context
-    found_dir
-      .strip_suffix('/')
-      .unwrap_or(found_dir)
-      .rfind('/')
-      .map(|i| &found_dir[..i + 1])
-      .unwrap_or(found_dir)
-  };
```

Comment:
Note this change. It doesn't include the parent name now in the file path because there's no really any need to do that and it improves the output to not have a single directory at the root.